addons/web/static/src/views/fields/statusbar/statusbar_field.scss

```diff
@@ -1,3 +1,20 @@
+.arrow-between-stages-before-after {
```

Comment:
You can use `Placeholder Selectors`[1] `%arrow-between-stages-before-after` instead of `.arrow-between-stages-before-after` as there no need to have the class itself in the file.

and IMO, you can use `mixin`/`include`[2] to avoid to post processing used by extend.

> Extends are resolved after the rest of your stylesheet is compiled.

[1]: https://sass-lang.com/documentation/at-rules/extend#placeholder-selectors
[2]: https://sass-lang.com/documentation/at-rules/mixin